spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-configuration-watcher-it/src/test/resources/kafka/kafka-deployment.yaml

```diff
@@ -33,9 +33,6 @@ spec:
             - name: KAFKA_LISTENER_SECURITY_PROTOCOL_MAP
               value: "INTERNAL:PLAINTEXT,OUTSIDE:PLAINTEXT"
 
-            - name: KAFKA_ZOOKEEPER_CONNECT
```

Comment:
this is incorrect and already present further down. it works as expected because of the order, still clean-up is needed here